transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/logging/Logging.java

```diff
@@ -27,7 +27,7 @@ private static String getTag(String tag) {
     if(android.os.Build.VERSION.SDK_INT < Build.VERSION_CODES.N)
       return concatTag(LOG_PREFIX, tag);
 
-    return MAX_LOG_TAG_SIZE_IN_SDK_N + tag;
+    return LOG_PREFIX + tag;
```

Comment:
I think this was unintended? otherwise also update the constant :-)

Reply:
Yup, sorry about that. I incorrectly used the constant for the limit instead of the prefix.

Reply:
I've updated the constant tag on the previous commit, sorry I created 2 commits.